aries-site/src/pages/learn/grid-fundamentals-part-one.mdx

```diff
@@ -0,0 +1,290 @@
+        rel="noopener noreferrer"
+        label="Heading"
+      />, <Anchor
+        href="https://v2.grommet.io/image"
+        target="_blank"
+        rel="noopener noreferrer"
+        label="Image"
+      />, <Anchor
+        href="https://v2.grommet.io/paragraph"
+        target="_blank"
+        rel="noopener noreferrer"
+        label="Paragraph"
+      />, and <Anchor
+        href="https://v2.grommet.io/text"
+        target="_blank"
+        rel="noopener noreferrer"
+        label="Text"
+      />.
+    </Paragraph>
+  }
+  width="large"
+  margin={{ bottom: 'medium' }}
+  status="learn"
+/>
+
+```js
+// App.js
+import { Grommet, Box, Heading, Text, ThemeContext, Grid } from 'grommet';
+import { hpe } from 'grommet-theme-hpe';
+import { ProductCard } from './ProductCard';
+import productList from './product-list.json';
+...
+```
+
+<br />
+
+Then, replace the placeholder content with the following:
+
+```js
+<Grid columns={{ count: 3, size: 'auto' }}>
+  {productList.map(product => (
+    <ProductCard
+      key={product.id}
+      name={product.name}
+      price={product.price}
+      image={product.image}
+    />
+  ))}
+</Grid>
+```
+
+Now the products are displayed in the grid with three columns.
+
+<StepTwo />
+
+## Responsive columns
+
+Take a moment to resize the browser window. Adjust the width, both wider and narrower. Notice
+that the products appear squished when the browser window is narrow and stretched when the
+browser window is wide. This is because the Grid is not responsive. Let's fix that.
+
+To make the Grid responsive, we will adjust the `columns` property. Set the `columns` property
+to `"medium"`.
+
+```js
+<Grid columns="medium">...</Grid>
+```
+
+Now the Grid will fit as many columns as will fit in the available space, with each column
+being a minimum width of the t-shirt size 'medium'.
+
+<Notification
+  message={
+    <Paragraph fill margin="none">
+      Did you know that Grommet uses "t-shirt sizes" to make implementing
+      consistent layouts easy? Learn more about{' '}
+      <Anchor href="/foundation/tshirt-sizing" label="t-shirt sizing" />.
+    </Paragraph>
+  }
+  width="large"
+  margin={{ bottom: 'medium' }}
+  status="learn"
+/>
+
+Notice how the number of columns changes as the browser window is resized. The Grid is now
+responsive.
+
+<StepThree />
+
+## Polishing the layout
+
+Now that we have a responsive Grid, let's polish the layout. Let's add some space
+between the products by adding `gap="small"` to the Grid.
+
+```js
+<Grid columns="medium" gap="small">
+  ...
+</Grid>
+```
+
+<StepFour />
+
+<br />
+
+Finally, let's refine the column width to be a bit more efficient with our screen's
+real-estate. A minimum column width of `"medium"` feels too wide. Let's try setting
+the `columns` property to `"small"`.
+
+```js
+<Grid columns="small" gap="small">
+  ...
+</Grid>
+```
+
+Now we are able to display more products in each row, but they become too narrow for my
```

Comment:
We tend not to use first person singular in our guidance. Can we adjust to ", but they become a bit too narrow." or something like that?

Reply:
done